cmd/kube-proxy/app/server.go

```diff
@@ -265,8 +269,11 @@ func (s *ProxyServer) Run() error {
 
 	s.Broadcaster.StartRecordingToSink(s.Client.Events(""))
 
-	// Start up Healthz service if requested
+	// Start up a webserver if requested
 	if s.Config.HealthzPort > 0 {
+		http.HandleFunc("/proxyMode", func(w http.ResponseWriter, r *http.Request) {
+			fmt.Fprintf(w, "%s", s.ProxyMode)
```

Comment:
More directly, this will say userspace if we don't have the right iptables version right?